examples/speech_to_text.v1.js

```diff
@@ -19,19 +27,20 @@ var recognizeStream = speechToText.createRecognizeStream(params);
 // pipe in some audio
 fs.createReadStream(__dirname + '/resources/speech.wav').pipe(recognizeStream);
 
-// and pipe out the transcription
+/*
+// pipe out the transcription to a file
 recognizeStream.pipe(fs.createWriteStream('transcription.txt'));
 
-// listen for 'data' events for just the final text
-// listen for 'results' events to get the raw JSON with interim results, timings, etc.
+// get strings instead of Buffers from `data` events
+// only do this when objectMode is `false`
```

Comment:
I'd take this line of the comment and move it to line 30.  You could then intro both lines of code with some reference to everything that's commented.  I think you want to call both `pipe` and `setEncoding` or neither of the two, right?

Reply:
That's a good idea, because they both apply to object mode being `false`. To clarify though, they do not have to be used together - you can set the encoding to log strings to the console without using the `pipe` statement to send it to a file. On the flip side, you can write the stream to a file without setting the encoding, but I imagine the user wouldn't find the results too helpful 🙂

Reply:
Thanks for the extra details, Dustin!